templates/bundle_version.sh

```diff
@@ -0,0 +1,16 @@
+GIT=$(sh /etc/profile; which git)
+GIT_RELEASE_VERSION=$(${GIT} describe --tags --always --abbrev=0)
+COMMITS=$(${GIT} rev-list HEAD | wc -l)
+COMMITS=$(($COMMITS))
+TARGET_PLIST="${TARGET_BUILD_DIR}/${INFOPLIST_PATH}"
+DSYM_PLIST="${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Info.plist"
+
+for PLIST in $TARGET_PLIST $DSYM_PLIST
+do
+  if [ -f "$PLIST" ] ; then
+    /usr/libexec/PlistBuddy -c "Set :CFBundleVersion ${COMMITS}" "$PLIST"
```

Comment:
Curly brances aren't required on `${COMMITS}`.

In general I don't mind when folks want to *always* use curly braces on all variables, but you should be consistent throughout the script, or only use them when required (like `${GIT_RELEASE_VERSION#*v}`)